src/fsharp/NameResolution.fs

```diff
@@ -1076,15 +1076,15 @@ let GetNestedTypesOfType (ad, ncenv:NameResolver, optFilter, staticResInfo, chec
 [<RequireQualifiedAccess>]
 type ItemOccurence = 
     /// This is a binding / declaration of the item
-    | Binding = 0
+    | Binding 
```

Comment:
This never needed to be a enum - better to make it a standard union type (as it is in FCS)